plenum/test/checkpoints/test_view_change_after_checkpoint.py

```diff
@@ -22,44 +22,55 @@ def sent_batches(request, chkFreqPatched):
     if request.param == 'equal_to_checkpoint':
         return CHK_FREQ
 
+low_watermark = 0
+batches_count = 0
 
+
+@pytest.mark.skip(reason="INDY-1336. For now, preprepares, prepares and commits queues are cleaned after view change")
```

Comment:
What exactly makes this test failing? It looks like the test is not dependent on the fact that 3PC messages are cleaned up now.